go-controller/pkg/metrics/ovn.go

```diff
@@ -342,6 +349,64 @@ func getPortCount(portType string) float64 {
 	return portCount
 }
 
+//ovnControllerSBDBConnectionCheckUpdater blocks until stopCh closed but before then polls ovn-controllers connection status with
+//southbound database periodically.
+func ovnControllerSBDBConnectionCheckUpdater(stopCh <-chan struct{}, ovsAppctl ovsClient, period time.Duration) {
+	// There maybe transient connection issues to SB DB. We want to minimise the risk of reporting this as the current state between
+	// long poll intervals.
+	retry := 5
+	retrySleep := 5 * time.Second
+	retryTotal := retrySleep * time.Duration(retry)
+
+	if retryTotal.Seconds() >= period.Seconds() {
```

Comment:
FYI: I think time.Duration can be compared directly

Reply:
Force pushed to remove the pointless call to convert it to seconds. Thanks @jcaamano